### PR TITLE
Adding soft_task_limit to kill the task that's hangs

### PR DESCRIPTION
**Pull request by awmackowiak**

### CODE REVIEW

File: vaas-app/src/vaas/settings/base.py [hunk 1]
```diff
@@ -169,6 +169,9 @@
 CELERY_IGNORE_RESULT = env.bool('CELERY_IGNORE_RESULT', False)
 CELERY_TASK_PUBLISH_RETRY = env.bool('CELERY_TASK_PUBLISH_RETRY', True)
 
+# 5min we will wait for kill task
+CELERY_TASK_SOFT_TIME_LIMIT = 300
```
Comment (TheMickeyMike) [inline vaas-app/src/vaas/settings/base.py]: I would suggest you, to rename this variable to `CELERY_TASK_SOFT_TIME_LIMIT_SECONDS`

File: vaas-app/src/vaas/manager/middleware.py [hunk 2]
```diff
@@ -88,6 +89,9 @@ def process_response(self, request, response):
 
                     if isinstance(result.result, Exception):
                         raise result.result
+            except SoftTimeLimitExceeded:
+                logging.info("Time for finish the task has been reached: The task with id {} will be killed.".format(
```
Comment (TheMickeyMike) [inline vaas-app/src/vaas/manager/middleware.py]: info? I think this should be logging.error

File: vaas-app/src/vaas/settings/base.py (hunk 1 above)
Comment (awmackowiak) [reply vaas-app/src/vaas/settings/base.py]: 👍

File: vaas-app/src/vaas/manager/middleware.py (hunk 2 above)
Comment (awmackowiak) [reply vaas-app/src/vaas/manager/middleware.py]: 👍

**Pull request merged**